index.d.ts

```diff
@@ -38,6 +38,10 @@ export interface FocusGroupKeyUXOptions {
 
 export type HotkeyOverride = Record<string, string>
 
+export interface Transformer {
+  (code: string, window: MinimalWindow, dir?: 'r'): false | string
```

Comment:
OK, let’s use argument. What do you think if we call it `reverse: boolean`?

Reply:
I tried both approaches, and in terms of compressed size two functions have a slight advantage. Since it's also aesthetically nicer, let's go that way? I updated the PR.